mono/mini/driver.c

```diff
@@ -928,7 +928,7 @@ test_thread_func (ThreadData *td)
 					guint pos = (*data)->start + random () % (*data)->length;
 					MonoJitInfo *ji;
 
-					ji = mono_jit_info_table_find (domain, (char*)(gulong) pos);
+					ji = mono_jit_info_table_find (domain, (char*)(uintptr_t)pos);
```

Comment:
`gsize` instead of `uintptr_t`

Reply:
I was thinking we should move away from the historical types. But maybe consistency and brevity says use them forever.

Reply:
Actually there seems to be some ambiguity as to if gsize==size_t or uintptr_t or the same thing. I claim size_t==uintptr_t but some people are less sure.

Reply:
Right I looked and we have gsize=uinptr_t for windows, else unsigned long, which on all normal platforms (except Win64) is pointer-size.